### ISSUE TITLE
gwpy incompatible with gwosc 0.7.0

### ISSUE THREAD

**Issue by GregoryAshton** [gwpy/gwpy#1605]
The latest version of `gwosc` break `TimeSeries.fetch_open_data`.

Script to reproduce:

```python
import gwpy
from gwpy.timeseries import TimeSeries
import gwosc

print(gwpy.__version__, gwosc.__version__)

time_of_event = 1126259462.4
post_trigger_duration = 2
duration = 4
analysis_start = time_of_event + post_trigger_duration - duration

H1_analysis_data = TimeSeries.fetch_open_data(
    "H1", analysis_start, analysis_start + duration)
```

Output 

```
3.0.3 0.7.0
Traceback (most recent call last):
  File "/home/greg/demo.py", line 12, in <module>
    H1_analysis_data = TimeSeries.fetch_open_data(
  File "/home/greg/anaconda3/envs/breaking/lib/python3.9/site-packages/gwpy/timeseries/core.py", line 476, in fetch_open_data
    return fetch_gwosc_data(
  File "/home/greg/anaconda3/envs/breaking/lib/python3.9/site-packages/gwpy/timeseries/io/losc.py", line 153, in fetch_gwosc_data
    cache = get_urls(detector, int(start), int(ceil(end)), **url_kw)
TypeError: get_urls() got an unexpected keyword argument 'tag'
```

**Comment by GregoryAshton** [comment]
I'd guess this has something to do with: https://git.ligo.org/gwosc/client/-/merge_requests/71?

**Comment by duncanmmacleod** [comment]
This will be fixed by #1606.
